modules/private-clusters-setting-api-private.adoc

```diff
@@ -37,48 +38,66 @@ ifndef::cpmso-using-azure[]
 *** For AWS, delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
 endif::cpmso-using-azure[]
 ifndef::cpmso-using-aws[]
-*** For Azure, delete the `api-internal` rule for the load balancer.
+*** For Azure, delete the `api-internal-v4` rule for the public load balancer.
+endif::cpmso-using-aws[]
+
+ifndef::cpmso-using-aws[]
+.. For the Azure private load balancer, create a rule called `outbound-rule-v4` that connects the private load balancer to the backend address pool.
```

Comment:
I am not sure about this, thanks @jinyunma helped to do many testing here, can you help to check if this is correct?

Reply:
Azure private load balancer can't have any outbound rules, here should be for Azure public load balancer, and rule named `outbound-rule-v4` is Outbound rule. 

@sunzhaohua2 and I did some tests for this scenario.

1. Created regular Azure IPI cluster, 3 inbound load balancer rules are created in public load balancer ( api-internal-v4 for port 6443, aa998da86225c46558762bb6403aeecc-TCP-80, aa998da86225c46558762bb6403aeecc-TCP-443)
2. Removed inbound lb rule "api-internal-v4", we found that cluster node can still communicate to internet, then tried to create outbound rule on azure portal, get error as below:
![Screenshot from 2024-04-25 11-40-14](https://github.com/openshift/openshift-docs/assets/61676078/5cb5daf0-aff8-4162-a0d4-7ef401d1f67c)

As ingress inbound lb rules still exists in public lb, they have implicit outbound rule enabled as described in the error, so there is no need to create explicit outbound rule any more.

3. We continue configuring the Ingress Controller endpoint publishing scope to Internal, two ingress inbound lb rules were removed automatically, we tested that cluster node could not communicate to internet until creating outbound rules in public load balancer.

So based on above testing, maybe outbound rule `outbound-rule-v4` for cluster backend address pool needs to be created only when there is no any inbound rules are present for same backend address pool in public load balancer.

cc @JoelSpeed @rna-afk to double confirm or any more options.

Reply:
@jinyunma thanks so much for explaining and running through the test. I made some changes in [this commit](https://github.com/openshift/openshift-docs/pull/73646/commits/f7412af78ec864b226f950fb62071f85f5bf7b54), let me know if I got it right :bow: